dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -21,7 +21,7 @@ export const menuOptions = [
         submenu: true,
         key: "all_runs",
         submenuOf: "dashboard",
-        link: "all_runs",
+        link: "all-runs",
```

Comment:
Why did you change the `link` value but not the `key` value?  (Should they be consistent?)

And, I second [Dave's comment](https://github.com/distributed-system-analysis/pbench/pull/3543/files#r1312929903):  if we're going to use the literal value in more than one place, we should define a constant and use that symbol.  Since we use the symbols in `dashboard/src/App.js` to define the routing, we should use them here as well, since these links are supposed to engage those routes.

Reply:
Updated